wasm/wasm_main.go

```diff
@@ -40,7 +51,11 @@ func jsEval(this js.Value, args []js.Value) interface{} {
 		anyErrs[i] = v
 	}
 	result["errors"] = anyErrs
-	result["formatted"] = strings.TrimSuffix(formatted, "\n")
+	fmt := strings.TrimSuffix(formatted, "\n")
```

Comment:
fmt is a package name, I wouldn't shadow it

Reply:
I recommend you enabling "import-shadowing" linter in revive.

Available via golangci-lint

Reply:
if you remember I have enable-all already so... it should show up

Reply:
oh so it's a non default rule of revive?

Reply:
not sure how to add 1 extra without having to specify all the other default, running it by hand once with:
```
linters-settings:
  revive:
    rules:
      - name: import-shadowing
```

Reply:
funny it found a few others (like errors and eval) but it doesn't mind this one :-)

Reply:
You can find the revive default ones in my 03-safe example here

https://github.com/ccoVeille/golangci-lint-config-examples

I have enabled them all and it helped me.

I still have to share a conf about it